### DISCUSSION TITLE
Pythonanywhere deployment

### DISCUSSION THREAD

**Discussion by MateoSaez** [elimintz/justpy#441] [Q&A]
Hi everyone.

Does someone tried to deploy a justpy project through pythonAnywhere?
I can't make the IP address binding part as it explained at: 
https://justpy.io/tutorial/deployment/

I get the next server error output:
ERROR:    [Errno 13] error while attempting to bind on address ('0.0.0.0', 80): permission denied

(Maybe it is an easy one issue but i get stacked)

Thanks in advance

Mateo

**Comment by elimintz** [comment]
PythonAnywhere does not currently support websockets. Maybe that is the issue. Another possibility is that you need to set the host and port. See https://justpy.io/reference/justpy/
